### DISCUSSION TITLE
Pinia in prod build

### DISCUSSION THREAD

**Discussion by ttonyh** [nuxt-alt/auth#86] [General]
Now that Pinia is optional, how do I exclude it from the bundle. After doing a build and checking the `entry.*.js` file, it seems to still be there. I made sure it was NOT in my nuxt config anymore. Maybe it's not possible to treeshake it since it's referenced in this auth lib?

**Comment by Denoder** [comment]
You're right. I think i'll make it a dynamic import to alleviate that and that'll make pinia get code split into another file. Will do that for the next version.
